notebook.ipynb

```diff
@@ -54,13 +54,13 @@
    "execution_count": null,
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        display(Markdown("You chooce to load the data from your local storage, you can skip the calls generation and move to [calls analysis](#calls_analysis)"))
</pre><p>Typo in the word chooce -&gt; choose</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demo-call-center/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>